tests/09-mdbook-mode/input/src/ch00-00-introduction.md

```diff
@@ -0,0 +1,5 @@
+# Introduction
+
+> Note: This edition of the book is the same as [The Rust Programming
+> Language][nsprust] available in print and ebook format from [No Starch
+> Press][nsp].
```

Comment:
These two links won't resolve. We do not do markdown link name thing across documents. In the original did you see blocks like:

```md
[nsprust]: https://nostarch.com/rust
[nsp]: https://nostarch.com/
```

?

If so you will have to include them also.

Reply:
Yes in original these blocks exists